.github/workflows/ant.yaml

```
@@ -107,20 +108,44 @@
           git tag "${{ steps.publish-lua-code.outputs.srcTxId }}"
           git push origin "${{ steps.publish-lua-code.outputs.srcTxId }}"
 
+      - name: Publish WASM Module
+        id: publish-module
+        env:
+          WALLET: ${{ secrets.WALLET }}
+          GITHUB_SHA: ${{ github.sha }}
+          DRY_RUN: ${{github.ref_name != 'main'}}
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)